lib/Arch/X86/Arch.cpp

```diff
@@ -838,8 +828,8 @@ static bool TryDecodeGetPCIdiom(Instruction &inst) {
   addr_op.type = Operand::kTypeAddressExpression;
```

Comment:
After decoding, you'll want to `inst.bytes.resize()` to `call_size + pop_inst.bytes.size()`.

Reply:
[95bfd8d](https://github.com/lifting-bits/remill/pull/610/commits/95bfd8d4ccf49ffa9ebbc92e69983aa5cc051abd)